clang/lib/Driver/ToolChains/AMDGPU.cpp

```diff
@@ -181,8 +181,15 @@ RocmInstallationDetector::getInstallationPathCandidates() {
 
     // Some versions of the rocm llvm package install to /opt/rocm/llvm/bin
```

Comment:
Doesn't this end up being the clang resource directory? Why do we need additional handling?